<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgProxSensDetectionParam</class>
 <widget class="QDialog" name="CQDlgProxSensDetectionParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detection Parameters</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QCheckBox" name="qqMinDistEnabled">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>271</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Don't allow detections if distance smaller than [m]</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqLimitedAngle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>271</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Limited angle detection, max. angle [deg]</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqBackFace">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Back face detection</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqOcclusionCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Occlusion check</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqFrontFace">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Front face detection</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqFast">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>231</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Fast detection (approximate)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqAngle">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqMinDist">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="aaaaa">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>115</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Randomized ray detection</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Ray count</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqRayCount">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Ray detections count for triggering</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqRandomizedDetectionCount">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="qqOkCancel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>190</y>
     <width>351</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
